src/gazebo_geotagged_images_plugin.cpp

```diff
@@ -578,12 +590,98 @@ void GeotaggedImagesPlugin::_handle_request_camera_settings(const mavlink_messag
         MAVLINK_COMM_1,
         &msg,
         0,                      // time_boot_ms
-        CAMERA_MODE_IMAGE,      // Camera Mode
+        _mode,                  // Camera Mode
         1.0E2 * (_zoom - 1.0)/ (_maxZoom - 1.0),                    // Zoom level
         NAN);                   // Focus level
     _send_mavlink_message(&msg, srcaddr);
 }
 
+void GeotaggedImagesPlugin::_handle_request_video_stream_status(const mavlink_message_t *pMsg, struct sockaddr* srcaddr)
+{
+    gzdbg << "Send videostream status" << endl;
+    mavlink_command_long_t cmd;
+    mavlink_msg_command_long_decode(pMsg, &cmd);
+    int sid = static_cast<int>(cmd.param1);
+    //-- Should we execute the command
+    if ((int)cmd.param1 != 1)
+    {
+        gzwarn << "Request ignored" << endl;
+        return;
+    }
+    // ACK command received and accepted
+    _send_cmd_ack(pMsg->sysid, pMsg->compid, MAV_CMD_REQUEST_VIDEO_STREAM_STATUS, MAV_RESULT_ACCEPTED, srcaddr);
+    mavlink_message_t msg;
+    mavlink_msg_video_stream_status_pack_chan(
+        1,
+        MAV_COMP_ID_CAMERA,                                     // Component ID
+        MAVLINK_COMM_1,
+        &msg,
+        static_cast<uint8_t>(sid),                              // Stream ID
+        VIDEO_STREAM_STATUS_FLAGS_RUNNING,                      // Flags (It's always running)
+        30,                                                     // Frame rate
+        _width,                                                  // Horizontal resolution
+        _height,                                                // Vertical resolution
+        2048,                                                   // Bit rate (made up)
```

Comment:
@Jaeyoung-Lim can we make the frame rate, bit rate, rotation and FOV configurable by SDF elements of the plugin? Makes more sent to me than hard-coding them.